### ISSUE TITLE
Dune build fails and takes ages to install

### ISSUE THREAD

**Issue by sh0tx420** [alexozer/flitter#26]
why do i have to install so much fucking dependencies, seriously, installing opam and all this shit took 1 hour and 30 minutes and now dune tells me to install more dependencies, can't you just compile this yourself and make it a release?

**Comment by sh0tx420** [comment]
File "vendor/notty/lwt/notty_lwt.ml", line 68, characters 25-64:
```
Error: This expression has type (unit -> unit) Lwt.t
       but an expression was expected of type unit Lwt.t
       Hint: Did you forget to provide `()' as argument?
Done: 216/230 (jobs: 1)^J
```

your code doesnt even compile lol

**Comment by alexozer** [comment]
so many*

**Comment by Cjnator38** [comment]
Yeah I have the same problem.Does this have anything to do with "Hint: Did you forget to provide `()' as argument?"
```
[christophersmith@this-is-pc flitter]$ dune build
File "_none_", line 1: 
Warning 58: no cmx file was found in path for module Toploop, and its interface was not compiled with -opaque
File "vendor/notty/lwt/notty_lwt.ml", line 68, characters 25-64:
Error: This expression has type (unit -> unit) Lwt.t
       but an expression was expected of type unit Lwt.t
       Hint: Did you forget to provide `()' as argument?
Done: 212/230 (jobs: 1)[christophersmith@this-is-pc flitter]$ dune build
File "vendor/notty/lwt/notty_lwt.ml", line 68, characters 25-64:
Error: This expression has type (unit -> unit) Lwt.t
       but an expression was expected of type unit Lwt.t
       Hint: Did you forget to provide `()' as argument?
[christophersmith@this-is-pc flitter]$
```

**Comment by alexozer** [comment]
Fixed in 666c5483bc93efa6d01e0b7a927461269f8e14de. And it's not a bad idea at all to provide a binary release.
